### PR TITLE
[GitHub#594] Allow logo to be updated in admin interface.

### PR DESCRIPTION
**Pull request by ivan-berezhnov**
Make sure these boxes are checked before asking for review of your pull request - thank you!

## General checks
- [ ] All coding styles are fulfilled and there are no any issues reported by CodeSniffer CI. <br/>CI code sniffer errors
- [ ] All tests are running and there are no failed tests reported by CI. <br/>Behat test results
- [ ] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [ ] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [ ] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [ ] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [ ] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

# Issues: 
Jira https://propeople-us.atlassian.net/browse/OYPD-525
GitHub https://github.com/ymcatwincities/openy/issues/594
Drupal.org issue: https://www.drupal.org/node/2892306

# Steps to review:
- [x] scroll down untill you can see footer
- [x] find logo in the footer
- [x] verify footer logo link have title "Site-logo"

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1805 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1805 |
| Installation process | http://install.openy-dev.ffwua.com/build1805/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1805/

**Comment by gundevel** [comment]
Decoupling report http://ci.openymca.org/build1805

**Comment by gundevel** [comment]
Accessibility report http://ci.openymca.org/build1805

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1805/profiles/contrib/openy/build/reports/behat

File: themes/openy_themes/openy_rose/templates/page/page--footer.html.html.twig [hunk 1]
```diff
@@ -64,7 +64,7 @@
         {% if footer_logo %}
           {% set background_image = " style='background-image: url(" ~ file_url(footer_logo) ~ ")'" %}
         {% endif %}
-        <a href="{{ front_page }}" class="footer__brand brand" title="{{ "YMCA"|t }}"{{ background_image|raw }}></a>
+        <a href="{{ front_page }}" class="footer__brand brand" title="{{ "Site-logo"|t }}"{{ background_image|raw }}></a>
```
Comment (alexschedrov) [inline themes/openy_themes/openy_rose/templates/page/page--footer.html.html.twig:67]: This change affects only footer for 3rd party services http://ci.openymca.org/build1805/footer.html.

We need to cover 3 other use cases:
- Footer in global template https://github.com/ymcatwincities/openy/blob/8.x-1.x/themes/openy_themes/openy_rose/templates/page/page.html.twig#L157
- Footer on camp page https://github.com/ymcatwincities/openy/blob/8.x-1.x/themes/openy_themes/openy_rose/templates/page/page--node--camp.html.twig#L45
- Landing page associated with Camp https://github.com/ymcatwincities/openy/blob/8.x-1.x/themes/openy_themes/openy_rose/templates/page/page--node--landing-page--has-camp.html.twig#L45

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by Sanchiz** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1810 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1810 |
| Installation process | http://install.openy-dev.ffwua.com/build1810/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1810/

**Comment by gundevel** [comment]
Decoupling report http://ci.openymca.org/build1810

**Comment by gundevel** [comment]
Accessibility report http://ci.openymca.org/build1810/htmlsniffers.md

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1810/profiles/contrib/openy/build/reports/behat

**Comment by Sanchiz** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1811 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1811 |
| Installation process | http://install.openy-dev.ffwua.com/build1811/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1811/

**Comment by gundevel** [comment]
Decoupling report http://ci.openymca.org/build1811

**Comment by gundevel** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://drupal.192.168.56.132.xip.io/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://drupal.192.168.56.132.xip.io/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://drupal.192.168.56.132.xip.io/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://drupal.192.168.56.132.xip.io/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://drupal.192.168.56.132.xip.io/blogWCAG2AAhtmlcs.txt
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build1811/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build1811/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build1811/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build1811/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build1811/blogWCAG2AAhtmlcs.txt

**Comment by Sanchiz** [comment]
retest this please

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1811/profiles/contrib/openy/build/reports/behat

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1812 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1812 |
| Installation process | http://install.openy-dev.ffwua.com/build1812/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1812/

**Comment by bjwilkins** [comment]
Decoupling report http://ci.openymca.org/build1812

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build1812/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build1812/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build1812/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build1812/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build1812/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Behat test results file URL: http://ci.openymca.org/build1812/profiles/contrib/openy/build/reports/behat

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1816 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1816 |
| Installation process | http://install.openy-dev.ffwua.com/build1816/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1816/

**Comment by bjwilkins** [comment]
Decoupling report http://ci.openymca.org/build1816

**Comment by bjwilkins** [comment]
Acessibility Sniffer: front page checking WCAG2AA http://ci.openymca.org/build1816/frontWCAG2AAhtmlcs.txt
Acessibility Sniffer: join page checking WCAG2AA http://ci.openymca.org/build1816/joinWCAG2AAhtmlcs.txt
Acessibility Sniffer: locations page checking WCAG2AA http://ci.openymca.org/build1816/locationsWCAG2AAhtmlcs.txt
Acessibility Sniffer: schedules page checking WCAG2AA http://ci.openymca.org/build1816/schedulesWCAG2AAhtmlcs.txt
Acessibility Sniffer: blog page checking WCAG2AA http://ci.openymca.org/build1816/blogWCAG2AAhtmlcs.txt

**Comment by bjwilkins** [comment]
Behat test results file URL: http://ci.openymca.org/build1816/profiles/contrib/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]

**Pull request merged**